dist/dialog.linker.build.js

```diff
@@ -71,17 +78,21 @@ window.jQuery = window.jQuery || window.shoestring;
 		this.$background.attr( "id", this.$el.attr( "id" ) + "-background" );
 		this._setBackgroundTransparency();
 
-		if( this.isSetScrollPosition() ) {
-			this.scroll = "pageYOffset" in w ? w.pageYOffset : ( docElem.scrollY || docElem.scrollTop || ( body && body.scrollY ) || 0 );
-			this.$el[ 0 ].style.top = this.scroll + "px";
-		} else {
-			this.$el[ 0 ].style.top = '';
-		}
```

Comment:
Is removing this related to the history change? 

It looks like we removed a feature, i.e. `data-set-position-media`. That might make it harder for projects update to this new version (which will likely be major).